ci-operator/jobs/openshift/cluster-policy-controller/openshift-cluster-policy-controller-master-presubmits.yaml

```diff
@@ -66,7 +66,7 @@ presubmits:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )e2e-aws,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
true or false? must've been manually updated, as this was generated.

Reply:
Yeah you likely want to copy the `jobs/` files first and edit the repo name there, then regenerate

Reply:
fixed, got it

Reply:
I'd go for copy only config files and regenerate jobs, there shouldn't be manual edits

Reply:
i don't have any manual edits, it's good